src/style_context.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use auto::StyleContext;
+use ffi;
+use gdk;
+use gdk_ffi;
+use glib::translate::*;
+use std::mem;
+use StateFlags;
+
+impl StyleContext {
+    pub fn get_color(&self, state: StateFlags) -> gdk::RGBA {
+        unsafe {
+            let mut ffi_rgba = mem::uninitialized();
+            ffi::gtk_style_context_get_color(self.to_glib_none().0,
+                                             state.to_glib(),
+                                             &mut ffi_rgba as *mut _);
+            from_glib_none(gdk_ffi::gdk_rgba_copy(&ffi_rgba))
+        }
+    }
+
+    pub fn lookup_color(&self, color_name: &str) -> Option<gdk::RGBA> {
```

Comment:
color_name is actually what's used in the GTK docs which is why I used it: https://developer.gnome.org/gtk3/stable/GtkStyleContext.html#gtk-style-context-lookup-color